### PR TITLE
Ignore sem ver extensions when comparing remote config values

### PR DESCRIPTION
**Pull request by elgatovital**
@gnosis/mobile-devs

### CODE REVIEW

File: app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt [hunk 1]
```diff
@@ -52,6 +65,43 @@ class SemVerTest {
...
+        val rangeString1 = "2.15.0...2.17.0"
+        val range1 = SemVer.parseRange(rangeString1, true)
+        assert(range1.first != null)
+        assert(range1.second != null)
+        assertEquals(version1, range1.first)
+        assertEquals(version2, range1.second)
+
+        val rangeString2 = "2.15.0"
+        val range2 = SemVer.parseRange(rangeString2, true)
+        assert(range2.first != null)
```
Comment (rmeissner) [inline app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt:84]: https://stackoverflow.com/questions/2966347/assert-vs-junit-assertions 😛

Comment (elgatovital) [reply app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt]: thank you for pointing this out

File: app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt [hunk 2]
```diff
@@ -18,6 +17,20 @@ class SemVerTest {
         assertEquals(SemVer(2, 15, 0, "255rc-internal"), version3)
     }
 
+    @Test
+    fun `parse (versionString, ignoreExtensions) should return SemVer`() {
+        val version1 = SemVer.parse("2.15.0", true)
+        assertEquals(SemVer(2, 15, 0), version1)
+
+        val version2 = SemVer.parse("2.15.0-255-internal", true)
+        assertNotEquals(SemVer(2, 15, 0, "255-internal"), version2)
```
Comment (biafra23) [inline app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt:26]: Why does this have to be not equal?

**Review by biafra23 — APPROVED** [review]

File: app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt (hunk 2 above)
Comment (elgatovital) [reply app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt:26]: yes. Here we test that extension part is ignored during parsing

**Pull request merged**